docs/sources/tutorials/alerting-get-started/alerting-get-started.md

```diff
@@ -0,0 +1,144 @@
+
+To add a link:
+
+1. In Title, enter Example.
+1. In the URL, enter https://example.com.
+1. Click Submit to add the link. The link will appear listed under the Grafana News heading.
+To vote for a link, click the triangle icon next to the name of the link.
+
+### Log in to Grafana
+
+Besides being an open-source observability tool, Grafana has its own built-in alerting service.   This means that you can receive notifications whenever there is an event of interest in your data, and even see these events graphed in your visualizations.
+
+1. In your browser, simply navigate to localhost:3000. You should get logged in automatically
+
+### Create the Alert
+
+Before we get started, it’s important to note that Grafana Alerting supports two alert rule types: Grafana-managed alert rules, which allow you to create alerts that can act on data from multiple data sources, and data source-managed alert rules, which are alert rules created and stored within the data source itself. In our case, our data source is Prometheus, so we will be creating a Grafana-managed alert rule. This is because Grafana uses Prometheus Alertmanager as the default Alertmanager, which handles alerts sent by client applications, such as the Prometheus server. Moreover, the exported metrics can be queried using the Prometheus Query language (PromQL)  
+
+1. In Grafana, toggle the menu at the top left side of the screen, and navigate to Alerting > Alert rules. Click on  + New alert rule.
+
+#### Enter alert rule name
+
+1. Give a name to the alert rule. For instance, server-requests-duration
+
+#### Define query and alert condition
+
+1. Choose the data source from the drop-down menu. Since our data source is Prometheus, the rule type should automatically switch to Grafana-managed alert. 
+
+
+1. In the Query editor, switch to Code mode
+P1. aste the query tns_request_duration_seconds_count{status_code="200",method="GET"}
+You should see the number of requests made to the server that had a response code 200.
+1. Keep expressions “B” and “C” as they are. These expressions (Reduce and Threshold, respectively) come by default when creating a new rule. Expression “B”, selects the last value of our query “A”, while the Threshold expression “C” will check if the last value from expression “B” is above a specific value. In addition, the Threshold expression is the alert condition by default. Enter 0.2 as threshold value. You can read more about queries and conditions here.
+1. Click Preview to run the queries.
+
+At this point, our alert should be working (it should be either in Firing or Normal state). If it returns an error, follow the instructions in the error message. If it returns “No data,” reselect the metrics from the Metrics browser; make sure the Input of each expression is correct (expression B: Input A and expression C: Input B). If you still get stuck, you are very welcome to post questions in our Grafana Community forum.
```

Comment:
nit: add a link to our community forums for when the user doesn't know where to find those?